http/translator.go

```diff
@@ -43,9 +41,9 @@ func NewTranslateStore(node interface{}) pilosa.TranslateStore {
 	case *pilosa.Node:
 		n = v
 	default:
-		log.Printf("WARNING: a *pilosa.Node is the only type supported by NewTranslateStore().")
```

Comment:
I don't think that this particular log is too problematic - it isn't polluting the output of Pilosa, and will only happen if someone's code is actually wrong. If I'm understanding this correctly it would not actually be unreasonable to panic here, although perhaps we should leave as a log to play it safe.

Reply:
Panic would  be more appropriate in reguards to maintaing the commitment to the released api, which philosophically i question in this scenerio, if we can bypass a panic situation with an alteration of an api, i feel its preferable to make the modification.  

Printing a log message returning a nil node in the case of an errant url would result in program failure as well, so no real difference in net result ie messge and program immediate termination.